README.md

```diff
@@ -180,8 +180,8 @@ Using a filter, you can easily enable/disable a plugin for a given environment.
         "hapi-pino": {
             "enabled": {
                 "$filter": "env.NODE_ENV",
-                "development": false,
-                "$default": true
+                "production": true,
+                "$default": false
```

Comment:
Checking against `production` instead since the absence of a value should imply development, not production.